nemo/collections/asr/data/audio_to_text.py

```diff
@@ -636,20 +637,34 @@ def __getitem__(self, item):
         pitch -= self.pitch_avg
         pitch[pitch == -self.pitch_avg] = 0.0  # Zero out values that were perviously zero
         pitch /= self.pitch_std
+        
+        speaker = None
+        if self.collection[item].speaker is not None:
+            speaker = torch.zeros_like(text_len).fill_(self.collection[item].speaker)
+        else:
+            # cannot return none as speaker, since it neural type validation fails.
+            speaker = torch.zeros_like(text_len).fill_(0)
```

Comment:
You can set `'speakers': NeuralType(('B',), Index(), optional=True),`

Reply:
done.